modules/core/04-channel/v2/keeper/packet.go

```diff
@@ -19,33 +20,26 @@ import (
 // in order for the packet to be sent to the counterparty.
 func (k *Keeper) sendPacket(
 	ctx context.Context,
-	sourceChannel string,
+	sourceId string,
 	timeoutTimestamp uint64,
 	payloads []types.Payload,
 ) (uint64, string, error) {
-	// Lookup channel associated with our source channel to retrieve the destination channel
-	channel, ok := k.GetChannel(ctx, sourceChannel)
-	if !ok {
-		// TODO: figure out how aliasing will work when more than one payload is sent.
-		channel, ok = k.convertV1Channel(ctx, payloads[0].SourcePort, sourceChannel)
-		if !ok {
-			return 0, "", errorsmod.Wrap(types.ErrChannelNotFound, sourceChannel)
-		}
+	// lookup counterparty and clientid from packet identifiers
+	clientID, counterparty, err := k.resolveV2Identifiers(ctx, payloads[0].SourcePort, sourceId)
```

Comment:
I thought this function was taking in a packet identifier? But it is taking in the source (client) identifier?